tests/system/providers/amazon/aws/example_ec2.py

```diff
@@ -142,6 +145,22 @@ def parse_response(instance_ids: list):
     )
     # [END howto_sensor_ec2_instance_state]
 
+    # [START howto_operator_ec2_reboot_instance]
+    reboot_instance = EC2RebootInstanceOperator(
+        task_id="reboot_instace",
+        instance_ids=instance_id,
+    )
+    # [END howto_operator_ec2_reboot_instance]
+    reboot_instance.trigger_rule = TriggerRule.ALL_DONE
```

Comment:
We don't need `TriggerRule.ALL_DONE` here (or on the hibernate), that's only for steps that are needed for cleanup, to ensure they run and don't leave behind resources. So it's only really needed on the terminate instance operator.

Reply:
Removed in https://github.com/apache/airflow/pull/35790/commits/57ad259b0b90d0ad8626e24f25061bca8947ddbf